bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xwing-miniatures-font",
-  "version": "2.3.2",
+  "version": "2.4.2",
```

Comment:
This should be `2.3.3` (I don't think it warrants a minor revision bump, as there's no change to the list of icons).

Also, `package.json` needs to be similarly updated (it's the more important file, as Bower is deprecated now).